notebooks/DynamoDbImportExport.ipynb

```diff
@@ -0,0 +1,495 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DynamoDB Import/Export Notebook\n",
+    "\n",
+    "Author: everett@bayesimpact.org\n",
+    "\n",
+    "Temporary solution to our need to back up (and possibly restore) dynamodb tables. Change the settings in the next cell, and then run the entire notebook.\n",
+    "\n",
+    "Good documentation on boto3, the python library for AWS APIs.\n",
+    "* [boto3.resource](http://boto3.readthedocs.io/en/latest/guide/dynamodb.html) (which you should use whenever possible)\n",
+    "* [boto3.client](http://boto3.readthedocs.io/en/latest/reference/services/dynamodb.html) (a more complete, but lower level API)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup: Configure this block, then run the whole notebook"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Set either RUN_IMPORT or RUN_EXPORT to true.\n",
+    "# Here we set both for the sake of example to see\n",
+    "# them both in action.\n",
+    "RUN_EXPORT = True\n",
+    "EXPORT_FILENAME = \"dynamodb.pickle\"\n",
+    "\n",
+    "RUN_IMPORT = True\n",
+    "IMPORT_FILENAME = \"dynamodb.pickle\"\n",
+    "IMPORT_TABLES_ADD_PREFIX = \"\"\n",
+    "\n",
+    "# AWS keys and DynamoDB region\n",
+    "# If these are left blank, we'll try to find environment\n",
+    "# variables by the same name and use those.\n",
+    "AWS_ACCESS_KEY_ID = ''\n",
+    "AWS_SECRET_ACCESS_KEY = ''\n",
+    "AWS_REGION = ''\n",
+    "\n",
+    "# If this is true, we connect to a dynamodb-local instance\n",
+    "# running on port 8000. The AWS keys above are then\n",
+    "# irrelevant, unless you changed it to not run as a --sharedDb.\n",
+    "USE_DYNAMODB_LOCAL = True"
```

Comment:
`USE_DYNAMODB_LOCAL` and `set_local_db()` are not used anywhere in the rest of the notebook.